<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="camera_sensor_model">
    <link name="link">
        <sensor type="camera" name="my_camera">
          <camera>
            <horizontal_fov>1.047</horizontal_fov>
            <image>
              <width>320</width>
              <height>240</height>
            </image>
            <clip>
              <near>0.1</near>
              <far>100</far>
            </clip>
          </camera>
          <always_on>1</always_on>
          <update_rate>4.0</update_rate>
          <visualize>true</visualize>
        </sensor>
    </link>
  </model>
</sdf>

